lib/widget/camera.dart

```diff
@@ -135,6 +158,23 @@ class CameraScreenState extends WidgetState<CameraScreen> with WidgetsBindingObs
     setState(() {});
   }
 
+  void defineErrorText() {
+    if (widget._controller.mode == CameraMode.photo) {
+      errorString =
+          'Maximum ${widget._controller.maxCount} images may be selected';
+      cameraoptionsList = ['PHOTO'];
+    } else if (widget._controller.mode == CameraMode.video) {
+      errorString =
+          'Maximum ${widget._controller.maxCount} videos may be selected';
+      cameraoptionsList = ['VIDEO'];
+    } else {
+      errorString =
+          'Maximum ${widget._controller.maxCount} images and videos may be selected';
+      cameraoptionsList = ['PHOTO', 'VIDEO'];
+    }
+    setState(() {});
```

Comment:
should not be calling setState in a function that is just constructing a message. fix your code please.